js/App.js

```diff
@@ -0,0 +1,68 @@
+var Socket = require('./utils/Socket'),
```

Comment:
I was asking about utils/Socket, I asked before you typed the rest of the update.